terraso_backend/apps/project_management/graphql/projects.py

```diff
@@ -173,6 +174,10 @@ class Input:
     def mutate_and_get_payload(cls, root, info, create_soil_settings=True, **kwargs):
         logger = cls.get_logger()
         user = info.context.user
+
+        if not check_project_permission(user, "create", Context()):
+            cls.not_allowed_create(Project)
```

Comment:
confirmation: this check always passes right now and is just here for completeness right? the site one can fail if attempting to create a site attached to a project without the right permissions, but this one doesn't have any failure conditions?

Reply:
Correct